<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DuSE::JavaScriptConsole</class>
 <widget class="QWidget" name="DuSE::JavaScriptConsole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>JavaScript Editor</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QTextEdit" name="txeJavaScript">
        <property name="acceptRichText">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QToolButton" name="tbtJSEvaluate">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Run (CTRL-R)</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="system-run">
            <normaloff/>
           </iconset>
          </property>
          <property name="shortcut">
           <string>Ctrl+R</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbtSaveScript">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Save script as</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="document-save">
            <normaloff/>
           </iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbtOpenScript">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Open script</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="document-open">
            <normaloff/>
           </iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>JavaScript Output</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QTextEdit" name="txeJavaScriptEvaluation">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="acceptRichText">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
